packages/remix-edge-adapter/src/common/server.ts

```diff
@@ -30,7 +34,8 @@ export function createRequestHandler({
 
   return async (request: Request, context: LoadContext): Promise<Response | void> => {
     const { pathname } = new URL(request.url)
-    // Skip the handler for static files
+    // Skip the handler for static files. We've already configured the generated SSR Edge Function
```

Comment:
nittiest of the nits: we only capitalise the name when referring to the product, not when referring to an instance of a function 🤓 

```suggestion
    // Skip the handler for static files. We've already configured the generated SSR edge function
```